packages/modular-scripts/src/build-scripts/webpack-scripts/config/parts/esmViewConfig.ts

```diff
@@ -1,18 +1,21 @@
-'use strict';
-const fs = require('fs-extra');
-const parse5 = require('parse5');
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-const InterpolateHtmlPlugin = require('../../../react-dev-utils/InterpolateHtmlPlugin');
-const { parsePackageName } = require('../utils/esmUtils');
-const dedent = require('dedent');
-const path = require('path');
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import dedent from 'dedent';
+import path from 'path';
+import webpack, { Configuration } from 'webpack';
+import { parsePackageName } from '../../../../utils/parsePackageName';
+import { rewriteModuleSpecifier } from '../../../../utils/buildImportMap';
+import fs from 'fs-extra';
+import parse5 from 'parse5';
+import { Element } from 'parse5/dist/tree-adapters/default';
+import { Paths } from '../../../common-scripts/determineTargetPaths';
```

Comment:
This two might be `import type`

Reply:
WebpackConfiguration and Configuration are the same from different libraries (webpack and webpack-dev-server)
will dedup